### PR TITLE
FOO-1331 fix packaging for native ESM environments

### PR DESCRIPTION
**Pull request by amireh**
This series of patches makes it possible to import `@instructure` packages found in this repository in environments where ES modules are natively supported, like Node 12.5+ and Webpack 5. Each part includes an explanation of the problem and the proposed solution in its commit message.

Thanks.

### CODE REVIEW

**Comment by amireh** [comment]
The last part is optional in that it is not part of the fixes, it only contains the tooling I used to verify the fixes in case you find it helpful too.

File: packages/ui-themes/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "author": "Instructure, Inc. Engineering and Product Design",
   "type": "module",
   "module": "./es/index.js",
-  "main": "./lib/index.js",
+  "main": "./es/index.js",
```
Comment (Brailor) [inline packages/ui-themes/package.json:8]: Just wondering why was this changed to the ES module source?

Comment (amireh) [reply packages/ui-themes/package.json:8]: There is no `lib/index.js`

**Review by Brailor — APPROVED** [review]
Great work, I really appreciated the detailed commit messages, approved from my side.

**Comment by amireh** [comment]
> Great work, I really appreciated the detailed commit messages, approved from my side.

Thank you Brailor

**Review by matyasf — APPROVED** [review]
thanks for this big contribution!
